test/js/pageobjects/CrossDeviceCheckYourMobile.js

```diff
@@ -0,0 +1,65 @@
+import Base from './BasePage.js'
+import {locale, verifyElementCopy} from '../utils/mochaw'
+
+class CrossDeviceCheckYourMobile extends Base{
+
+  get crossDeviceCheckYourMobileTitle() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
+  get crossDeviceCheckYourMobileSubmessage() { return this.$('.onfido-sdk-ui-crossDevice-MobileNotificationSent-submessage')}
+  get crossDeviceCheckYourMobileMayTakeFewMinutesMessage() { return this.$('.onfido-sdk-ui-crossDevice-MobileNotificationSent-boldMessage')}
+  get crossDeviceCheckYourMobilePhoneIcon() { return this.$('.onfido-sdk-ui-Theme-icon')}
+  get crossDeviceCheckYourMobileTipsHeader() { return this.$('.onfido-sdk-ui-Theme-header')}
+  get crossDeviceCheckYourMobileTipsFirst() { return this.$('li:nth-child(1)')}
+  get crossDeviceCheckYourMobileTipsSecond() { return this.$('li:nth-child(2)')}
```

Comment:
Since you are actually checking all of the list items' text between lines 45-53, you could just loop through each `<li>` element found and verify each one's text against an array of the expected text strings of the list.

Reply:
Done.